python/dgl/data/chem/csv_dataset.py

```diff
@@ -10,6 +10,12 @@
 from .utils import smile2graph
 
 
+try:
+    import pandas as pd
+    import rdkit
+except ImportError:
+    pass
+
 class CSVDataset(object):
```

Comment:
CSVDataset -> MoleculeCSVDataset. In the docstring, would you mind describing a little bit about the schema of the input CSV?

Reply:
Sure.